haskell/test/Stablecoin.hs

```diff
@@ -26,11 +29,23 @@ import Tezos.Core
 contractPath :: FilePath
 contractPath = "test/resources/stablecoin.tz"
 
-spec_Stablecoin :: Spec
-spec_Stablecoin = specWithUntypedContract contractPath specCallback
+origination :: forall param . U.Contract -> OriginationFn param
+origination contract (mkInitialStorage -> Just storageVal) =
+  -- We just cast the stablecoin contract address as an FA2 contract
```

Comment:
This comment looks wrong now. We cast to abstract `param` now, not to FA2 parameter.

Reply:
This comment still looks wrong.